core/controllers/base_test.py

```diff
@@ -712,6 +713,51 @@ def test_controller_class_names(self):
         self.assertGreater(num_handlers_checked, 150)
 
 
+class ValueGeneratorNameTests(test_utils.GenericTestBase):
```

Comment:
Let's put this in core/domain/value_generators_domain_test.py

Reply:
Done